WooCommerce/WooCommerceTests/ViewRelated/Products/BottomSheetListSelector/ProductsSortOrderBottomSheetListSelectorCommandTests.swift

```diff
@@ -4,16 +4,25 @@ import XCTest
 
 final class ProductsSortOrderBottomSheetListSelectorCommandTests: XCTestCase {
     func testInitialSelectedValue() {
+        // Arrange
         let selected = ProductsSortOrder.nameAscending
-        let command = ProductsSortOrderBottomSheetListSelectorCommand(selected: selected)
+        var selectedActions = [ProductsSortOrder]()
+        let command = ProductsSortOrderBottomSheetListSelectorCommand(selected: selected) { (selected) in
+            selectedActions.append(selected)
+        }
+        // Action
+        command.handleSelectedChange(selected: .nameAscending)
+        // Assert
```

Comment:
Since we're only testing that the initial value is set and **we never modify it after that**, a lot of these can be removed. 

```suggestion
        // Arrange
        let selected = ProductsSortOrder.nameAscending

        // Action
        let command = ProductsSortOrderBottomSheetListSelectorCommand(selected: selected) { (selected) in
            // noop
        }

        // Assert
        XCTAssertEqual(command.selected, selected)
```

- The `selectedActions` is not tested so we don't need to keep it
- The `command.handleSelectedChange(selected: .nameAscending)` has [no effect aside from calling the callback](https://github.com/woocommerce/woocommerce-ios/blob/871b9910e5178d39800a11c23af4168f6c55ad8e/WooCommerce/Classes/ViewRelated/Products/BottomSheetListSelector/ProductsSortOrderBottomSheetListSelectorCommand.swift#L46-L48) so it doesn't really affect the value of `command.selected`. 

The shorter the test, the easier it is to understand. 😊